theme/cobalt2.json

```diff
@@ -372,7 +372,7 @@
       "name": "Meta",
       "scope": "meta",
       "settings": {
-        "foreground": "#9effff"
+        "foreground": "#fb94ff"
```

Comment:
Based on my understanding, I recommend not modifying this scope, as it is designed to be applicable to all programming languages. However, if you require a distinct scope specifically for Python, I suggest creating a new name and scope for that purpose. This would ensure that your Python-specific modifications do not affect the functionality of other programming languages. Something like:

```json
{
  "name": "[PYTHON] - Meta",
  "scope": [
    "meta.function-call.arguments.python",
    "meta.function-call.python",
    "meta.member.access.python"
  ],
  "settings": {
    "foreground": "#fb94ff",
  }
},
```

I have not tested this, but see if it works.

Reply:
Sorry, my bad firstly i get it wrong 
Please check: commit 2676634c2e7e39ec3ac99b00e1427f19e826ac06
i fix that make new scope for python only for other language remains same.
output same as before
![image](https://user-images.githubusercontent.com/125372245/231085848-62694691-7212-483d-b0ca-5958b71d726d.png)